From 69bc3f53732a27ef66b4c0109c3a7d3f4f7bf735 Mon Sep 17 00:00:00 2001
From: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
Date: Tue, 30 Jan 2024 17:31:38 +0530
Subject: [PATCH] crypto: qce - Reduce priority for skcipher algo

Reduce priority for skcipher algo for qce driver.
with this qce driver will bind with specific use case
of crypto operation.

Change-Id: I5619da94555d31545fa8e172693dde39a1fb64a0
Signed-off-by: Poovendhan Selvaraj <quic_poovendh@quicinc.com>
(cherry picked from commit d9a5e006bf1fd056ba24a939620731adfe5f5db3)
---
 drivers/crypto/qce/skcipher.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/qce/skcipher.c b/drivers/crypto/qce/skcipher.c
index 9ab44e80180e..0e8754195a65 100644
--- a/drivers/crypto/qce/skcipher.c
+++ b/drivers/crypto/qce/skcipher.c
@@ -495,7 +495,7 @@ static int qce_skcipher_register_one(const struct qce_skcipher_def *def,
 	alg->encrypt			= qce_skcipher_encrypt;
 	alg->decrypt			= qce_skcipher_decrypt;
 
-	alg->base.cra_priority		= 300;
+	alg->base.cra_priority		= 10;
 	alg->base.cra_flags		= CRYPTO_ALG_ASYNC |
 					  CRYPTO_ALG_ALLOCATES_MEMORY |
 					  CRYPTO_ALG_KERN_DRIVER_ONLY;
-- 
2.34.1

